lib/_http_client.js

```diff
@@ -65,6 +65,14 @@ function isInvalidPath(s) {
   return false;
 }
 
+function validateHost(host) {
+  if (host !== undefined && host !== null && typeof host !== 'string') {
```

Comment:
The first two conditionals can be shortened to just `host != null` or `host != undefined`.